corehq/apps/app_manager/forms.py

```diff
@@ -55,3 +55,9 @@ def clean_domain(self):
         if domain is None:
             raise forms.ValidationError("A valid project space is required.")
         return domain_name
+
+    def clean(self):
+        domain = self.cleaned_data.get('domain')
+        if self.cleaned_data.get('linked'):
+            if not LINKED_APPS.enabled(domain):
+                raise forms.ValidationError("The target project space does not have linked apps enabled.")
```

Comment:
We wanted to hide the ability to make them behind a feature flag, so it was necessary for the master domain. Since the other domain was the one that actually has the linked app, and it radically alters the UI and behavior of the app manager, it seemed important to directly turn it on there too, but I can see the argument for only having it on the master domain.